modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts

```diff
@@ -53,7 +54,7 @@ const renderContextToolbar = (spec: { onEscape: () => Optional<boolean>; sink: A
       AddEventsBehaviour.config('context-toolbar-events', [
         AlloyEvents.runOnSource<EventArgs<TransitionEvent>>(NativeEvents.transitionend(), (comp, se) => {
           if (se.event.raw.propertyName === 'top' || se.event.raw.propertyName === 'bottom') {
-            Class.remove(comp.element, contextBarTransitionClass);
+            // Class.remove(comp.element, contextBarTransitionClass);
```

Comment:
Is the class still being removed by something else ? Or was this not meant to be commented out, and instead changed?